src/modules/campath/common.rs

```diff
@@ -0,0 +1,35 @@
+use nom::bytes::complete::tag;
+use nom::character::complete::space0;
+use nom::combinator::{map, opt, recognize};
+use nom::number::complete::float;
+use nom::sequence::preceded;
+use nom::IResult;
+
+#[derive(Clone, Copy)]
+pub struct ViewInfo {
+    pub vieworg: [f64; 3],
```

Comment:
If that helps, we should have Vec3 and similar types, they probably have lerp on them and stuff.